bin/replication.js

```diff
@@ -105,6 +110,73 @@ program.version(version);
         });
 });
 
+program
+    .command('multi-site-setup')
+    .description('if a Zenko site is included as a replication endpoint for a' +
+        ' one-to-many setup, "target-bucket" and "target-profile" are needed')
+    .option('--source-bucket <name>', '[required] source bucket name')
+    .option('--source-profile <name>',
+            '[required] source aws/credentials profile')
+    .option('--target-bucket <name>', '[optional] necessary if adding a zenko' +
```

Comment:
Following the other descriptions maybe this should read as '[optional] target bucket name (necessary if adding a Zenko site as a replication endpoint)